composer.json

```
@@ -1,10 +1,55 @@
 {
-    "name"       : "automattic/msm-sitemap",
-    "description": "Comprehensive sitemaps for your WordPress.com VIP site.",
-    "homepage"   : "https://vip.wordpress.com/plugins/msm-sitemap/",
-    "type"       : "wordpress-plugin",
-    "license"    : "GPL-2.0+",
-    "require"    : {
-        "composer/installers": "~1.0"
+  "name": "automattic/msm-sitemap",
+  "description": "Comprehensive sitemaps for your WordPress.com VIP site.",
+  "homepage": "https://github.com/Automattic/msm-sitemap/",
+  "type": "wordpress-plugin",
+  "license": "GPL-2.0+",
```

```suggestion
  "license": "GPL-2.0-or-later",
```

GPL-2.0+ isn't a valid SPDX identifier.